Dockerfile

```diff
@@ -1,68 +1,28 @@
-FROM peenuty/rails-passenger-nginx-docker-i:1.0.1
+FROM golang:1.9
 MAINTAINER Mike Rapadas <mike@mrap.me>
 
-# Install Node
+COPY . /go/src/github.com/mrap/combo
 
-    # GET NODE INSTALL DEPS
-    RUN       apt-get update --fix-missing
-    RUN       apt-get install -y build-essential curl libssl-dev
+WORKDIR /go/src/github.com/mrap/combo
 
-    # INSTALL NODE
-    RUN       curl -sL https://deb.nodesource.com/setup_6.x | bash
-    RUN       apt-get install nodejs
+RUN curl -sL https://deb.nodesource.com/setup_6.x | bash && \
+  curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - && \
+  echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list && \
+  apt-get update && \
+  apt-get install -y nodejs yarn && \
```

Comment:
We should group commands with `&&` that are alike.  So that a small change doesn't require a full rebuild.